### PR TITLE
chore(contracts): remove fast and safe conf levels

### PR DESCRIPTION
**Pull request by kevinhalliday**
Remove fast and safe conf levels from ConfLevel lib.

issue: #1329

### CODE REVIEW

File: contracts/core/src/libraries/ConfLevel.sol [hunk 1]
```diff
@@ -5,24 +5,15 @@ pragma solidity ^0.8.12;
...
  * @dev We prefer explicit constants over Enums, because we want uint8 values to start at 1, not 0, as they do in
- *      lib/xchain/types.go, such that 0 can represent "unset".
+ *      lib/xchain/types.go, such that 0 can represent "unset". Note only latest and finalized levels are supported
+ *      on-chain.
  */
 library ConfLevel {
     /**
      * @notice XMsg confirmation level "latest", last byte of xmsg.shardId.
      */
     uint8 internal constant Latest = 1;
```
Comment (ttarsi) [inline contracts/core/src/libraries/ConfLevel.sol:15]: can we maybe use an enum for this?

File: contracts/core/src/libraries/ConfLevel.sol [hunk 2]
```diff
@@ -5,24 +5,15 @@ pragma solidity ^0.8.12;
...
-     * @notice XMsg confirmation level "fast", last byte of xmsg.shardId.
-     */
-    uint8 internal constant Fast = 2;
-
-    /**
-     * @notice XMsg confirmation level "safe", last byte of xmsg.shardId.
-     */
-    uint8 internal constant Safe = 3;
-
     /**
```
Comment (ttarsi) [inline contracts/core/src/libraries/ConfLevel.sol:17]: latest and finalized to 1 & 2?

**Review by ttarsi — APPROVED** [review]
approving with Qs

File: contracts/core/src/libraries/ConfLevel.sol (hunk 2 above)
Comment (kevinhalliday) [reply contracts/core/src/libraries/ConfLevel.sol:17]: They are still 1 & 4 in lib/xchain/types.go

File: contracts/core/src/libraries/ConfLevel.sol (hunk 1 above)
Comment (kevinhalliday) [reply contracts/core/src/libraries/ConfLevel.sol:15]: Enum are zero indexed and we start with one. And they always increment montonically. So the translation to lib/xchain/types.go values is just a little annoying. I'd rather just set constants explicitly

File: contracts/core/src/libraries/ConfLevel.sol (hunk 2 above)
Comment (kevinhalliday) [reply contracts/core/src/libraries/ConfLevel.sol:17]: If we change them there I'm down

**Pull request merged**